doc/en/cache.rst

```diff
@@ -13,17 +13,21 @@ Cache: working with cross-testrun state
 Usage
 ---------
 
-The plugin provides two command line options to rerun failures from the
+The plugin provides command line options to rerun failures from the
 last ``py.test`` invocation:
 
 * ``--lf`` (last failures) - to only re-run the failures.
 * ``--ff`` (failures first) - to run the failures first and then the rest of
   the tests.
+* ``--lf-keep`` (last full failure set) - will prevent partial clearning
```

Comment:
This text seems confusing to me... perhaps this could be written as:

```
* ``--lf-keep`` (last full failure set) - will not clear the previous set of failed tests, so a test 
will only be removed from the failed set if it passes. This works particularly well when used in combination 
with ``--ff --lf-keep -x``, so tests can be fixed one at a time.
```